### PR TITLE
Pubsub fix

### PR DESCRIPTION
**Pull request by mbogoevici**

### CODE REVIEW

File: spring-cloud-stream-samples/source/src/main/resources/application.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ spring:
   cloud:
     stream:
       bindings:
-        output: testtock
+        output: topic:testtock
```
Comment (markfisher) [inline spring-cloud-stream-samples/source/src/main/resources/application.yml]: did you leave this intentionally?

Comment (mbogoevici) [inline spring-cloud-stream-samples/source/src/main/resources/application.yml]: No. Needs to be reverted.

Comment (mbogoevici) [inline spring-cloud-stream-samples/source/src/main/resources/application.yml]: We need actual binding tests for this, though.

**Comment by markfisher** [comment]
squashed and merged as https://github.com/spring-cloud/spring-cloud-stream/commit/b8ffa630f73621b3e1ae72b0111e1019565adfb5

**Pull request closed without merging**